### PR TITLE
lecture 1 for genomics course

### PR DESCRIPTION
**Pull request by nekrut**
A draft PR for new course to be based entirely on GTN. Need to get this accepted for a lecture tomorrow

### CODE REVIEW

**Review by shiltemann — APPROVED** [review]

File: topics/data-science/tutorials/gnmx-lecture1/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+subtopic: gnmx
+draft: true
+---
+
+![Fluctuation test](./images/luria_small.png)
+
+# What do you know?
+
+This is the first lecture of the class. We will begin with a short quiz that is designed tio let me know what you do and do not know. This quiz will have no effect on your grade in this course.
```
Comment (hexylena) [inline topics/data-science/tutorials/gnmx-lecture1/tutorial.md:25]: would you be willing to share the quiz? we have a built in quiz feature but it's probably not ready for prime-time, but we might be able to make use of it later.

**Comment by shiltemann** [comment]
(ok, just noticed that draft tutorials don't show up in draft learning pathways, will fix)

**Comment by shiltemann** [comment]
the direct link for the first lecture is here: 
- tutorial: https://training.galaxyproject.org/topics/data-science/tutorials/gnmx-lecture1/tutorial.html
- slides: https://training.galaxyproject.org/topics/data-science/tutorials/gnmx-lecture1/slides.html

**Comment by shiltemann** [comment]
ok, the learning pathway is now fixed to display the draft materials: https://training.galaxyproject.org/training-material/learning-pathways/data-driven-biology.html

If we keep adding your draft materials to this learning pathways your students will only need this one link.

Good luck with the course!

File: topics/data-science/tutorials/gnmx-lecture1/tutorial.md (hunk 1 above)
Comment (nekrut) [reply topics/data-science/tutorials/gnmx-lecture1/tutorial.md:25]: sure. once I make it ;)

**Pull request merged**